<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="2,1" columnstretch="2,1" rowminimumheight="2,1" columnminimumwidth="2,1">
    <item row="0" column="1">
     <widget class="QFrame" name="state">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="random">
         <property name="text">
          <string>Random</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="compute">
         <property name="text">
          <string>Compute</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="show">
         <property name="text">
          <string>Show</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="chess">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="0">
        <widget class="QFrame" name="place_09">
         <property name="styleSheet">
          <string notr="true">border-color: rgb(0, 0, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="text_09">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>87</width>
            <height>87</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 30pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QFrame" name="place_10">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="text_10">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>87</width>
            <height>87</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 30pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QFrame" name="place_11">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="text_11">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>87</width>
            <height>87</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 30pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFrame" name="place_02">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="text_02">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>87</width>
            <height>87</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 30pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QFrame" name="place_12">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="text_12">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>87</width>
            <height>87</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 30pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QFrame" name="place_07">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="text_07">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>87</width>
            <height>87</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 30pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QFrame" name="place_04">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="text_04">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>87</width>
            <height>87</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 30pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QFrame" name="place_01">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="text_01">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>87</width>
            <height>87</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="place_05">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="text_05">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>87</width>
            <height>87</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 30pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QFrame" name="place_03">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="text_03">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>87</width>
            <height>87</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 30pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QFrame" name="place_08">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="text_08">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>87</width>
            <height>87</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 30pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QFrame" name="place_13">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="text_13">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>87</width>
            <height>87</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 30pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QFrame" name="place_14">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="text_14">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>87</width>
            <height>87</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 30pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QFrame" name="place_15">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="text_15">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>87</width>
            <height>87</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 30pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QFrame" name="place_16">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="text_16">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>87</width>
            <height>87</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 30pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QFrame" name="place_06">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="text_06">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>87</width>
            <height>87</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 30pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="order">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3">
       <item>
        <widget class="QLabel" name="solve_label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="answer">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTextEdit" name="inputs"/>
       </item>
       <item>
        <widget class="QPushButton" name="commit">
         <property name="text">
          <string>commit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
